src/pages/elements/color/tokens.mdx

```diff
@@ -8,6 +8,83 @@ tabs: ['Overview', 'Usage', 'Tokens', 'Code']
 
 import ColorTokenTable from 'components/ColorTokenTable';
 
-## Tokens by theme
+<PageDescription>
 
-<ColorTokenTable />
+Tokens are a method of applying color in a consistent, reusable, and scalable
+way. They are used in place of hard coded values, like hex codes.
+
+</PageDescription>
+
+<AnchorLinks>
+
+<AnchorLink>Core Tokens</AnchorLink>
+<AnchorLink>Component Tokens</AnchorLink>
+<AnchorLink>AI Tokens</AnchorLink>
+
+</AnchorLinks>
+
+## Core Tokens
+
+Core tokens are global colors that are used across components. They are named
+and grouped by the common UI element that they are applied to.
+
+<AnchorLinks>
+
+<AnchorLink>Background</AnchorLink>
```

Comment:
These anchor links under the H2 sections are coming in at the larger size. Can we make them smaller like the anchor link on the [2x Grid](https://carbondesignsystem.com/elements/2x-grid/overview/#2x-grid-fundamentals) page?